src/us/kbase/auth2/kbase/KBaseAuthConfig.java

```diff
@@ -150,14 +151,16 @@ public KBaseAuthConfig(final Path filepath, final boolean nullLogger)
 		return Collections.unmodifiableSet(ips);
 	}
 	
-	private Map<String, String> getCustom(final String keyprefix, final Map<String, String> cfg) {
-		final Map<String, String> ret = new HashMap<>();
+	private void addCustom(
+			final Builder idProviderConfig,
+			final String keyprefix,
```

Comment:
I think `stripkeyprefix` would make the intent a bit clearer but that's a quibble.

Reply:
The string is used primarily to find the keys in the `if` statement, not just stripping the prefix from the key, so I don't think labelling the purpose as "strip" is accurate, really.